wandb/sdk/launch/launch.py

```diff
@@ -103,21 +103,12 @@ def run(
     synchronous: Optional[bool] = True,
 ) -> AbstractRun:
     """
-    Run a W&B project. The project can be local or stored at a Git URI.
-    W&B provides built-in support for running projects locally or remotely on a Databricks or
-    Kubernetes cluster. You can also run projects against other targets by installing an appropriate
-    third-party plugin. See `Community Plugins <../plugins.html#community-plugins>`_ for more
-    information.
-    For information on using this method in chained workflows, see `Building Multistep Workflows
-    <../projects.html#building-multistep-workflows>`_.
+    Run a W&B launch experiment. The project can be wandb uri or a Git URI.
     :raises: :py:class:`wandb.exceptions.ExecutionException` If a run launched in blocking mode
              is unsuccessful.
-    :param uri: URI of project to run. A local filesystem path
-                or a Git repository URI pointing to a project directory containing an MLproject file.
-    :param entry_point: Entry point to run within the project. If no entry point with the specified
-                        name is found, runs the project file ``entry_point`` as a script,
-                        using "python" to run ``.py`` files and the default shell (specified by
-                        environment variable ``$SHELL``) to run ``.sh`` files.
+    :param uri: URI of experiment to run. A wandb run uri or a Git repository URI.
+    :param api: An instance of a wandb Api from wandb.apis.internal.
+    :param entry_point: Entry point to run within the project. Defaults to using the command used to run the original run for wandb URIs, or main.py for git repository URIs.
     :param version: For Git-based projects, either a commit hash or a branch name.
     :param parameters: Parameters (dictionary) for the entry point command.
```

Comment:
do we also default to the same parameters as pulled from the original run command if nothing is provided? i actually don't remember

Reply:
yeah, I'll add that